### ISSUE TITLE
User Experience Survey 2022 Analysis and Sharing

### ISSUE THREAD

**Issue by Katrina-Birch** [alphagov/govuk-design-system#2084]
## What
Analyse and share results from Design System User Experience Survey 2022
## Why
To inform planning and prioritisation of future research, share the good stuff, identify quick wins to improve the contributor experience, identify and consider how the team might approach consistent themes.
## Who needs to work on this
User Researcher, Performance Analyst, wider team.
## Who needs to review this
Product Manager
## Done when
Research is analysed, shared and we have some objectives to inform future work
- [x] Quantitative data analysed and shared with team
- [x] Qualitative data analysed with the team 
- [ ] Pain points / unmet needs are feed back into the prioritisation process

**Comment by lfdebrux** [comment]
Blocked on time.

**Comment by Katrina-Birch** [comment]
I have passed the survey results to Performance Analyst, will chase up.

**Comment by kellylee-gds** [comment]
@Katrina-Birch I've moved this into blocked for the time being as I'm not sure Trang and myself can look at this until DSDay is done
